### PR TITLE
Disable loadout for plasmoids (BUGFIX...?)

### PR DESCRIPTION
**Pull request by cyfause**
## About The Pull Request
This disables the ability for plasmemes to spawn with loadout items.

## Why It's Good For The Game
It fixes a fringe case bug where if a plasmaman spawns in with no envirosuit/helmet/plasma gas canister - if they have a uniform, hat, or hand-held items selected in their loadout.
People finding this out on their own would probably be quite unhappy if they didn't know about it previously, so disabling the loadout entirely for plasmamen is something quick and sloppy to counteract it.

ufoj5fEO8Q

(The way that the code is written for loadout stuff makes it very hard to specifically disable uniform/hand/helmet cosmetics, so maybe if someone else knows how to do that please help oh god PLEASE)

## Changelog
:cl:
fix: disable loadout for plasmoids
/:cl:

### CODE REVIEW

**Comment by Dahlular** [comment]
you gave plasmapeople penises but you didn't give them scarves. I am proud.

File: code/controllers/subsystem/job.dm [hunk 1]
```diff
@@ -416,7 +416,7 @@ SUBSYSTEM_DEF(job)
 		H.mind.assigned_role = rank
 
 	if(job)
-		if(!job.dresscodecompliant)// CIT CHANGE - dress code compliance
+		if(!job.dresscodecompliant && !/datum/species/plasmaman)// CIT CHANGE - dress code compliance
```
Comment (LetterN) [inline code/controllers/subsystem/job.dm]:
```suggestion
		if(!job.dresscodecompliant && !isplasmaman(M))// CIT CHANGE - dress code compliance
```

Comment (LetterN) [reply code/controllers/subsystem/job.dm]: `/datum/species/plasmaman` shouldn't have worked at all!

Comment (cyfause) [reply code/controllers/subsystem/job.dm]: i am not good with computer and i dont know how i got here

Comment (cyfause) [reply code/controllers/subsystem/job.dm]: image
i tested your change and it stopped my thing from working *scream

Comment (LetterN) [reply code/controllers/subsystem/job.dm]: check line 444 and line 446, change `if(job.dresscodecompliant)` to `if(job.dresscodecompliant && !isplasmaman(M))` and add a if(!isplasmaman(M)) on line 446

**Comment by Dahlular** [comment]
been unresolved for too long

**Pull request closed without merging**